docs/analysis/index.rst

```diff
@@ -138,35 +134,23 @@ The observations are stored as a list of `~gammapy.data.DataStoreObservation` ob
 Data reduction and datasets
 ---------------------------
 
-The data reduction process needs a choice of a dataset type, declared as the class name
-(`~gammapy.cube.MapDataset`, `~gammapy.spectrum.SpectrumDatasetOnOff`) in the ``reduction`` section of the settings. For the
-estimation of the background with a dataset type `~gammapy.spectrum.SpectrumDatasetOnOff`, a ``background_estimator``
-is needed, other parameters related with the ``on_region`` and ``exclusion_mask`` FITS file
-may be also present. Parameters for geometry are also needed and declared in this section,
-as well as a boolean flag ``stack-datasets``.
+The data reduction process needs a choice of a dataset type, declared as ``1d`` or ``3d``
+in the ``datasets`` section of the settings. For the estimation of the background in a ``1d``
+use case, a background ``method`` is needed, other parameters related like the ``onregion``
```

Comment:
Suggest to rename `onregion` -> `on_region` in the config file.
We already have `analysis.background_estimator.on_region`, and we use `snake_case` almost consistently in the config file (except `binsize`, but that word we'll probably re-discuss anyways).

Reply:
Done.